components/tensorboard-controller/controllers/tensorboard_controller.go

```diff
@@ -246,6 +246,12 @@ func generateDeployment(tb *tensorboardv1alpha1.Tensorboard, log logr.Logger, r
 		})
 	}
 
+	// copy all of the CR labels to the pod which includes poddefault related labels
+	podLabels := map[string]string{"app": tb.Name}
```

Comment:
@surajkota could you initially set this as an empty dict and then set the `app: tb.Name` *after* the for loop?

We want to make sure the Controller will be correctly setting this value, even if a user requested a different value

Reply:
Done, please review. Thanks